public/index.html

```diff
@@ -24,7 +24,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Juntos</title>
+    <script src="https://apis.google.com/js/api.js"></script>
```

Comment:
I [read](https://stackoverflow.com/questions/51977448/how-to-use-gapi-in-react) there won't be any performance issue if this is added to public/index.html.

Let me know if you still want me to look into the lazy loading thing for the script.

Reply:
https://stackoverflow.com/questions/51977448/how-to-use-gapi-in-react doesn't talk about performance.

Let's add a comment here saying: "Needed in SelectVideo for Youtube video search"

So that we can take up lazy loading later.

Reply:
> Also, don't worry about it slowing anything down or loading the script before you need it.

This was mentioned, so I didn't try lazy loading the script.

Reply:
Needed in SelectVideo for YouTube video search

Reply:
I meant let's add a code comment in `index.html`

> Also, don't worry about it slowing anything down or loading the script before you need it.

I don't think this is accurate the file is > 5 Kb gzipped and > 12 KB overall